src/DetailsView/components/report-export-dialog-factory.tsx

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
```

Comment:
Was just curious about this change since this is a file that I don't think interacts with this feature!

Reply:
Added this because the props type is an extension of CommandBarProps and assumed that featureFlagStoreData was added to the props through that. Once I removed featureFlagStoreData from CommandBarProps, it no longer existed as part of ReportExportDialogFactoryProps. So that's why I added it here

Reply:
That makes so much sense, gracias!